litezip/completezip.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+
+from .main import parse_litezip
+
+
+__all__ = (
+    'convert_completezip',
+)
+
+
+def convert_completezip(path):
+    """Converts a completezip file structure to a litezip file structure.
+    Returns a litezip data structure.
+
+    """
+    for filepath in path.glob('**/index_auto_generated.cnxml'):
```

Comment:
unfortunately it's not this simple. The auto_generated is the correct file to keep, if the CNXML version < 0.7 inside the index.cnxml.